src/components/StateListItem.vue

```diff
@@ -51,10 +51,11 @@
     value?: unknown,
     stateType?: StateType | null | undefined,
     tags?: string[] | null,
+    disabled?: boolean,
   }>()
 
   const component = computed(() => {
-    if (props.value !== undefined) {
+    if (!props.disabled && props.value !== undefined) {
```

Comment:
curious about the `value !=== undefined` piece. Do we need that if disabled controls whether or not the checkbox is shown?

Reply:
Oh that should be an `||` danget, good eye.

Reply:
Do we even need the undefined check? Or is was that existing and we're just preserving that functionality.

Reply:
Re in slack: updated again. Now it supports the empty array for selected as another way to disable it.. Not great, but fixes it for now. Really tough to fallow these props since they're forwarded as attrs from far up the chain in some places, so this is the safest bet for now.